src/commands/build.ts

```diff
@@ -17,6 +17,7 @@ export class BuildCommand {
 
   async run(path = './') {
     const pod = new Pod(fs.realpathSync(path));
+    await pod.setup();
```

Comment:
Hmm... is there any way this could be updated to avoid the need to call `setup`? In other use cases (i.e. if an app or external code was written to hook into the pod), we don't want them to have to also remember to call `setup`.

Reply:
The main issue I was having was that the `import` is a Promise. I was thinking of using a `Promise.all` instead which would load the plugins independently, but the plugin order would need to be preserved since the order can be important when the plugin triggers happen. I'll update it when I have some time so that it doesn't need the async `setup`.

Reply:
Removed the async pod setup in favor of doing a sync `require` in the pod constructor.